src/poll.rs

```diff
@@ -556,7 +563,7 @@ pub struct Events {
 }
 
 /// Iterate an Events structure
-pub struct EventsIter<'a> {
+pub struct Iter<'a> {
```

Comment:
This naming typically works best when the `poll` module itself is public because we can't lift multiple `Iter` types into the root namespace. For `std::collections`, for example, the collections are all lifted to that module but all iterators live inside `std::collections::$relevant_module::Iter` (for example)

Making `poll` public may be somewhat difficult though